FCCee/CLD/compact/README.md

```diff
@@ -33,8 +33,13 @@ This model is based on `FCCee_o2_v02` from k4geo. It has an updated design of th
 standard design (low-impedance, no HOM absorber); changed Vertex Detector Barrel layers to fit the beampipe contraints (reduced barrel length); fixed
 Overlaps in the Inner and Outer Tracker.
 
+CLD_o2_v06
+----------
+
+This model is based on `CLD_o2_v05` with the following changes: LumiCal outer radius from 112 mm to 115 mm, remove its
+nose shield, place it at 1074*mm along the beam-axis, instead of at 1074*mm in Z, move LumiCal by 100*micron
```

Comment:
I had removed the comment about removing the nose shield because it looks like it was actually never there (just some dimensions defined in CLD xml but not used)

Reply:
Yes, good point. I changed the comment to refer to the removal of unused parameters.